metrics/ingestion_index_health_tracker.go

```diff
@@ -0,0 +1,71 @@
+package metrics
+
+import (
+	"context"
+	"time"
+
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/flow-evm-gateway/services/requester"
+)
+
+type IngestionIndexHealthTracker struct {
+	collector           Collector
+	evm                 *requester.EVM
+	logger              zerolog.Logger
+	maxHeightDifference uint64
+	interval            time.Duration
+}
+
+func NewIngestionIndexHealthTracker(logger zerolog.Logger, collector Collector, evm *requester.EVM, maxHeightDiff uint64, interval time.Duration) *IngestionIndexHealthTracker {
+	return &IngestionIndexHealthTracker{
+		collector:           collector,
+		evm:                 evm,
+		logger:              logger,
+		maxHeightDifference: maxHeightDiff,
+		interval:            interval,
+	}
+}
+
+func (c *IngestionIndexHealthTracker) Start(ctx context.Context) {
+	go func() {
+		ticker := time.NewTicker(c.interval)
+		defer ticker.Stop()
+
+		for {
+			select {
+			case <-ctx.Done():
+				c.logger.Info().Msg("shutting down block height collector")
+				return
+			case <-ticker.C:
+				err := c.updateIndexHealth(ctx)
+				if err != nil {
+					c.logger.Error().Err(err).Msg("error updating index health")
+				}
+			}
+		}
+	}()
+}
+
+func (c *IngestionIndexHealthTracker) updateIndexHealth(ctx context.Context) error {
+	indexedHeight, err := c.collector.EvmBlockIndex()
```

Comment:
I think this could all done a bit simpler, so instead of reporting the index health as 1/0, we could just report the latest EVM network height, and then we can implement the health boolean in grafana itself, since we have all the data there